Makefile

```diff
@@ -106,46 +93,22 @@ production_aks:
 .PHONY: qa
```

Comment:
just for my understanding, why are we not also removing these?

Reply:
Good question. It's mentioned on the last bullet point in the changes proposed in this pull request, but essentially we still need the DEPLOY_ENV=qa etc for the DNS management.